src/vs/platform/terminal/node/terminalProcess.ts

```diff
@@ -63,9 +66,10 @@ export class TerminalProcess extends Disposable implements ITerminalChildProcess
 
 	private _isPtyPaused: boolean = false;
 	private _unacknowledgedCharCount: number = 0;
-
 	public get exitMessage(): string | undefined { return this._exitMessage; }
+
 	public get currentTitle(): string { return this._currentTitle; }
+	public get lastTitle(): string { return this._windowsShellHelper?.lastShellTitle || this._currentTitle; }
```

Comment:
Would this work?

```ts
public get currentTitle(): string {
  return this._windowsShellHelper?.shellTitle || this._currentTitle
}
```

This feels more correct to me since `currentTitle` isn't very useful on Windows right now since `WindowsShellHelper` fills in that functionality.